packages/@lwc/integration-karma/test-hydration/light-dom/scoped-styles/deduped-scoped-styles/index.spec.js

```diff
@@ -0,0 +1,9 @@
+export default {
+    test(target, snapshot, consoleCalls) {
+        // Expect no errors or warnings, hydration or otherwise
```

Comment:
```suggestion
        // W-19087941: Expect no errors or warnings, hydration or otherwise
```
This is a pretty small test and it's not obvious why it exists, so we should try to keep the context around.